sdk/src/main/java/io/radar/sdk/model/RadarSdkConfiguration.kt

```diff
@@ -50,6 +52,7 @@ internal data class RadarSdkConfiguration(
                 config.optBoolean(START_TRACKING_ON_INITIALIZE, false),
                 config.optBoolean(TRACK_ONCE_ON_APP_OPEN, false),
                 config.optBoolean(USE_LOCATION_METADATA, false),
+                config.optBoolean(USE_OPENED_APP_CONVERSION, false),
```

Comment:
Should default true (log first opened app prior to getting anything synced down)